app/themes/user-admin/e2e/user-list/user-list.e2e-spec.ts

```diff
@@ -0,0 +1,115 @@
+import { UserAdminPage } from './user-list.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('User List Page', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
+        page.navigateTo();
+        let email = browser.findElement(protractor.By.name('email'));
+        let password = browser.findElement(protractor.By.name('password'));
+        let submit = browser.findElement(protractor.By.className('btn btn-success'));
+
+        email.sendKeys('foo@mail.com');
+        password.sendKeys('asdf1234');
+        
+        submit.click();
+
+        expect(page.getParagraphText("app-home h1")).toEqual('Welcome to the Erdiko User Admin');
+    });
+
+    it('should lead to User List page at click of the User List link', () => {
+        //check User Link in Nav bar
+        let userListNav = browser.findElement(protractor.by.css('nav ul > li:nth-child(2) > a'));
+        userListNav.click();
+
+        //go back to home page
+        let home = browser.findElement(protractor.by.css('nav ul > li:first-child > a'));
+        home.click()
+
+        //check User Link in Body
+        let userListBody = browser.findElement(protractor.by.css('app-home ul > li:nth-child(2) > a'));
+        userListBody.click();
+    });
+
+    it('should show the user list table header', () => {
+
+        //check for table header
+        expect(page.getParagraphText("thead th:first-child")).toEqual('ID');
```

Comment:
Is it possible to query for elements by their text values? e.g. 'Find element with innerHTML of ID' and test for truthfulness